ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1095,10 +1199,12 @@ function createBaseSkillsAttributes() {
       ...props,
     });
   }
-  addNewDiscipline('People');
-  addNewSkill('Language(Common)', { skillattribute: 'IQ', skillexpertise: 'ST' });
-  addNewSkill('Persuade', { skillattribute: 'IQ', skillexpertise: 'ST' });
-  addNewSkill('People Insight', { skillattribute: 'IQ', skillexpertise: 'ST' });
+  if (! skip_personal) {
```

Comment:
Linting should normalize this.
```suggestion
  if (!skip_personal) {
```